### ISSUE TITLE
Remove any HTML in Markdown docs

### ISSUE THREAD

**Issue by ankudinov** [aristanetworks/ansible-avd#60]
HTML has to be removed from Markdown docs. It's not critical, but increases complexity of maintaining and compatible with all dev tools.  
Here is an example:
```markdown
## Role Inputs and Outputs

Figure 1 below provides a visualization of the roles inputs, and outputs and tasks in order executed by the role.

<p align="center">
  <img src='media/figure-1-role-eos_l3ls_evpn.gif' alt='Figure 1: Ansible Role eos_l3ls_evpn'/>
</p>

**Inputs:**
```  
This is not displayed correctly in PyCharm for instance.

A simple Markdown string instead would be much better:
```markdown
![Figure 1: Ansible Role eos_l3ls_evpn](media/figure-1-role-eos_l3ls_evpn.gif)
```

**Comment by ankudinov** [comment]
Same for Fig 2.

**Comment by carlbuchmann** [comment]
resolved with: 5037df18c65e3a4a0987583c30ac9b6bc28e4507
